packages/teams-js/src/internal/telemetry.ts

```diff
@@ -99,6 +109,36 @@ export enum ApiName {
   Media_ScanBarCode = 'media.scanBarCode',
   Media_SelectMedia = 'media.selectMedia',
   Media_ViewImages = 'media.viewImages',
+  Meeting_AppShareButton_SetOptions = 'meeting.appShareButton.setOptions',
+  Meeting_GetAppContentStageSharingCapabilities = 'meeting.getAppContentStageSharingCapabilities',
+  Meeting_GetAppContentStageSharingState = 'meeting.getAppContentStageSharingState',
+  Meeting_GetAuthenticationTokenForAnonymousUser = 'meeting.getAuthenticationTokenForAnonymousUser',
+  Meeting_GetIncomingClientAudioState = 'meeting.getIncomingClientAudioState',
+  Meeting_GetLiveStreamState = 'meeting.getLiveStreamState',
+  Meeting_GetMeetingDetails = 'meeting.getMeetingDetails',
+  Meeting_RegisterAudioDeviceSelectionChangedHandler = 'meeting.registerAudioDeviceSelectionChangedHandler',
+  Meeting_RegisterLiveStreamChangedHandler = 'meeting.registerLiveStreamChangedHandler',
+  Meeting_RegisterMeetingReactionReceivedHandler = 'meeting.registerMeetingReactionReceivedHandler',
+  Meeting_RegisterMicStateChangeHandler = 'meeting.registerMicStateChangeHandler',
+  Meeting_RegisterRaiseHandStateChangedHandler = 'meeting.registerRaiseHandStateChangedHandler',
+  Meeting_RegisterSpeakingStateChangeHandler = 'meeting.registerSpeakingStateChangeHandler',
+  Meeting_RequestAppAudioHandling = 'meeting.requestAppAudioHandling',
+  Meeting_RequestStartLiveStreaming = 'meeting.requestStartLiveStreaming',
+  Meeting_RequestStopLiveStreaming = 'meeting.requestStopLiveStreaming',
+  Meeting_ShareAppContentToStage = 'meeting.shareAppContentToStage',
+  Meeting_StopSharingAppContentToStage = 'meeting.stopSharingAppContentToStage',
+  Meeting_ToggleIncomingClientAudio = 'meeting.toggleIncomingClientAudio',
+  Meeting_SetMicStateWithReason = 'meeting.setMicStateWithReason',
+  Menus_HandleActionMenuItemPress = 'menus.handleActionMenuItemPress',
+  Menus_HandleNavBarMenuItemPress = 'menus.handleNavBarMenuItemPress',
+  Menus_HandleViewConfigItemPress = 'menus.handleViewConfigItemPress',
+  Menus_RegisterActionMenuItemPressHandler = 'menus.registerActionMenuItemPressHandler',
+  Menus_RegisterNavBarMenuItemPressHandler = 'menus.registerNavBarMenuItemPressHandler',
+  Menus_RegisterSetModuleViewHandler = 'menus.registerSetModuleViewHandler',
+  Menus_SetUpViews = 'menus.setUpViews',
```

Comment:
>Menus_SetUpViews [](http://example.com/codeflow?start=2&length=16)

This seems like it should alphabetically be below `Menus_SetNavBarMenu`

Reply:
fixed.